chains/rari-arbitrum-sepolia-testnet.json

```diff
@@ -0,0 +1,23 @@
+{
+  "alias": "rari-arbitrum-sepolia-testnet",
+  "decimals": 18,
+  "explorer": {
+    "api": {
+      "key": {
+        "required": false
+      },
+      "url": "https://testnet.explorer.rarichain.org/api"
+    },
+    "browserUrl": "https://testnet.explorer.rarichain.org/"
+  },
+  "id": "1918988905",
+  "name": "Rari Arbitrum Sepolia testnet",
```

Comment:
The hardest part of reviewing these PRs are checking and deciding naming. It shouldn't be subjective but it makes me feel so.

In my opinion, it should be named as `RARI Arbitrum Sepolia testnet` because everywhere it is presented uppercased. 

Same applies for its [mainnet](https://github.com/api3dao/chains/blob/6a5f3c1272a96d23ef364645dfdb2e0fa865b363/chains/rari.json) definition. 

Additionally, I'm not sure about adding `Chain` which results in `RARI Chain Arbitrum Sepolia testnet`. See mainnet counterpart it has `Chain`.